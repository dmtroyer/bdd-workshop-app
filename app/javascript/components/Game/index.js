import React from 'react';
import WarGamesIntro from '../WarGamesIntro';

const Game = () => <WarGamesIntro />;

export default Game;
