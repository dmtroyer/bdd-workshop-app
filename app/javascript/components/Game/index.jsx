import React from 'react';
import { Link } from 'react-router-dom';
import { Button } from 'reactstrap';

const Game = () => (
  <div data-testid="game">
    <div data-testid="game-status">Please create a profile first!</div>
    <Button color="primary" tag={Link} to="/profile">
      Create Profile NOW!
    </Button>
  </div>
);

export default Game;
