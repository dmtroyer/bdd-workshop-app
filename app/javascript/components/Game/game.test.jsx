import React from 'react';
import { shallow } from 'enzyme';
import Game from '.';

describe('Game', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Game />);
    expect(wrapper).toMatchSnapshot();
  });
});
