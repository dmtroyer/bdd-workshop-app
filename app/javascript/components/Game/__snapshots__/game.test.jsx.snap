// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game renders correctly 1`] = `
<div
  data-testid="game"
>
  <div
    data-testid="game-status"
  >
    Please create a profile first!
  </div>
  <Button
    color="primary"
    tag={[Function]}
    to="/profile"
  >
    Create Profile NOW!
  </Button>
</div>
`;
